Consistently use GetWidth() to fix disappeared timers

karupayun opened the pull request:
In https://github.com/google/orbit/commit/44ec55c7942ef1dae7a6f6363451c40e0ce3e85e, we started to use GetWidth() instead of viewport->ScreenWidth() for the width of the tracks, since we were no longer considering the vertical slider.

However there were still a few missing references we didn't change, which
caused a strange behaviour in ThreadTrack. So, in this PR we erase them.

Test: Load a capture.
Bug related: http://b/196873688.

Pd: There are still a few references in AccessibleCaptureViewElement
which I'm not sure how to solve it.

florian-kuebler reviewed (approved):
LGTM. Also tested that, and the "disappearing timers" issue is solved. However, still the height issue remains.

karupayun commented:
> LGTM. Also tested that, and the "disappearing timers" issue is solved. However, still the height issue remains.

It's not the same Tori fixed yesterday? Sorry, I didn't rebase to the latest main before. Now it's rebased.

reichlfl reviewed (commented):
Thanks for fixing this! I suggest to use `viewport_->WorldToScreenWidth(GetWidth())` to convert from world- to pixel coordinates though - technically, it doesn't make a difference because we're not (yet) scaling the world width, but this is done everywhere else for consistency, and will be important once we replace the world- with point-space.

pierricgimmig reviewed (approved):
Cool, thanks for the fix @karupayun

karupayun commented:
> Thanks for fixing this! I suggest to use `viewport_->WorldToScreenWidth(GetWidth())` to convert from world- to pixel coordinates though - technically, it doesn't make a difference because we're not (yet) scaling the world width, but this is done everywhere else for consistency, and will be important once we replace the world- with point-space.

Thanks for pointing this, I replaced this ones and also a few that I replaced before in https://github.com/google/orbit/commit/44ec55c7942ef1dae7a6f6363451c40e0ce3e85e

reichlfl reviewed (approved):
LGTM, thanks for the changes!
